src/Pages/License/Generate/GenerateLicense.jsx

```diff
@@ -4,11 +4,14 @@ import { styled } from "@mui/styles";
 import CardContent from "@mui/material/CardContent";
```

Comment:
Separate groups for react, material ui and internal components. Put line breaks between each group

Reply:
done